src/prefect/artifacts.py

```diff
@@ -14,6 +16,27 @@ def _running_with_backend() -> bool:
     return bool(context.get("running_with_backend"))
 
 
+def _create_task_run_artifact(kind: str, data: dict) -> str:
+    client = Client()
+    task_run_id = context.get("task_run_id")
+    # XXX: there's a race condition in the cloud backend for mapped tasks where
```

Comment:
XXX?

Reply:
I was wondering the same thing lol

Reply:
https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

I have my editor setup to highlight these comments, drawing attention to them as things that could be better (but aren't really todos).